runtime/interpreter/sharedstate.go

```diff
@@ -44,6 +44,7 @@ type SharedState struct {
 	CapabilityControllerIterations              map[AddressPath]int
 	MutationDuringCapabilityControllerIteration bool
 	containerValueIteration                     map[atree.StorageID]struct{}
+	resourceDestruction            map[atree.StorageID]struct{}
```

Comment:
not formatted properly: linter might complain

Reply:
Fixed in 4eb4b537997cab5ea2f5e9fabd35e0ca2835842f